lmnet/lmnet/data_augmentor.py

```diff
@@ -29,7 +29,7 @@ class Blur(data_processor.Processor):
     """Gaussian blur filter.
 
     Reference:
-        http://pillow.readthedocs.io/en/4.3.x/Referenceserence/ImageFilter.html#PIL.ImageFilter.GaussianBlur
+        https://pillow.readthedocs.io/en/stable/reference/ImageFilter.html#PIL.ImageFilter.GaussianBlur
```

Comment:
I thinks document of current version is better than latest `stable` version.
Current using version's documentation is not avairable?

Reply:
As far as I know, Blueoil's current master branch is using 

[`Pillow==6.0.0`](https://github.com/blue-oil/blueoil/blob/9fcdbfc8808ef8f4509e0c4ae0729b30c298e943/lmnet/requirements.txt#L6)

However, if you select a specific version of the documentation, it doesn't exist anymore. The selection of documentation is either the latest or stable release version.

Reply:
As far as I know, Blueoil's current master branch is using

`Pillow==6.0.0`

However, if you select a specific version on the documentation, it doesn't exist anymore. The selection of documentation is either the latest or stable release version.

So selecting version 6.0.0 is kind of impossible. As it is not active in the docs.

https://readthedocs.org/projects/pillow/versions/

Reply:
@Joeper214 Oh, thanks.
We should update Pillow...